konflux-ci/ui/core/chrome/chrome-frontend.yaml

```diff
@@ -79,6 +79,10 @@ spec:
               name: config
             - mountPath: /opt/app-root/src/build/preview/operator-generated
               name: config
+            - mountPath: /opt/app-root/src/build/stable/js/d6083bde9a964ab4.svg
+              subPath: d6083bde9a964ab4.svg
+              name: logo
+              readOnly: true
```

Comment:
Is this line needed?
Doesn't `readOnlyRootFilesystem: true` under `securityContext` specifies that the entire root filesystem of the container should be mounted as read-only ?

Reply:
I'm pretty sure that `readOnlyRootFilesystem: true` applies only to the files bundled in the container image, not mounted files.